source/includes/package-repository/_40-update.md.erb

```diff
@@ -0,0 +1,98 @@
+## Update a package repository
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/repositories/0530f1a9-5282-4b69-be59-066a35076452' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v1+json' \
+      -H 'Content-Type: application/json' \
+      -X PUT -d '{
+        "name" : "npmjs.org",
+        "plugin_metadata": {
+          "id": "npm",
+          "version": "1"
+        },
+        "configuration": [
+          {
+            "key": "REPO_URL",
+            "value": "http://registry.npmjs.com"
+          },
+          {
+            "key": "USERNAME",
+            "value": "admin"
+          },
+          {
+            "key": "PASSWORD",
+            "encrypted_value": "aSdiFgRRZ6A="
+          }
+        ]
+      }'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.go.cd.v1+json; charset=utf-8
+ETag: "4167e3ec81fdac0fb29d854b36ceb981"
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/repositories/0530f1a9-5282-4b69-be59-066a35076452"
+    },
+    "doc": {
+      "href": "https://api.go.cd/#package-repository"
+    },
+    "find": {
+      "href": "https://ci.example.com/go/api/admin/repositories/:repo_id"
+    }
+  },
+  "repo_id": "0530f1a9-5282-4b69-be59-066a35076452",
+  "name": "npmjs.org",
+  "plugin_metadata": {
+    "id": "npm",
+    "version": "1"
+  },
+  "configuration": [
+    {
+      "key": "REPO_URL",
+      "value": "http://registry.npmjs.com"
+    },
+    {
+      "key": "USERNAME",
+      "value": "admin"
+    },
+    {
+      "key": "PASSWORD",
+      "encrypted_value": "aSdiFgRRZ6A="
+    }
+  ],
+  "packages": [
```

Comment:
Can you make it clear that users can edit the repo configuration for a repository which has some packages?